Shared/GeneralUtils.swift

```diff
@@ -4,6 +4,43 @@
 
 import Foundation
 
+// Wraps NimbleDroid to ensure it is disabled in release
+// Use underscores between words, as these constants are stringified for reporting.
+public struct Profiler {
+    public enum Bookend {
+        case bvc_did_appear
+        case url_autocomplete
+        case intro_did_appear
+        case history_panel_fetch
+        case load_url
+        case find_in_page
+    }
+
+    public static func setup() {
+        if AppConstants.BuildChannel != .release {
```

Comment:
Is it not better to check if we are running in the nimbledroid test environment instead? This would for example also enable this code in the `.beta` BuildChannel, which is what runs via TestFlight on end user devices. I think we need to narrow this down more to just when we need it.

Reply:
It prints useful profiling info as the app runs. I thought this would be useful for developers (maybe not so useful from Beta audience thought).